modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSCSSNodeSassPatternsCheck.java

```diff
@@ -98,7 +103,7 @@ else if (!interpolationPart.isEmpty()) {
 	}
 
 	private static final Pattern _divisionPattern = Pattern.compile(
-		"(\\$\\w+|[0-9.]+)\\s*\\/\\s*(\\$\\w+|[0-9.]+)");
+		"(\\$\\w+|[0-9]+[.]*[0-9]+)\\s*\\/\\s*(\\$\\w+|[0-9]+[.]*[0-9]+)");
```

Comment:
@nicolas-sss, shouldn't be `[0-9]+[.]*[0-9]*` instead of `[0-9]+[.]*[0-9]+`?

If I understood it correctly then we may have math divisions using `1` and `1.1`, for example, consequently, if we've `.`, in theory, we must also have a number after it as in `1.1`. Does it make sense to you?

Reply:
Yes, it should. Good call!